src/components/Card/Card.tsx

```diff
@@ -4,15 +4,17 @@ import styles from './Card.module.scss'
 
 export interface CardProps extends ComponentProps<'div'> {
   rounded?: boolean | 'top' | 'bottom'
+  shadow?: boolean
```

Comment:
It is ambiguous I think, because there is another meaning for shadow, `boxShadow` instead maybe better.

Reply:
I'm not quite sure what you mean by the other meaning of `shadow`?

In my opinion, `shadow` referring to `box-shadow` is a common implementation, as seen in Tailwind CSS as well.

However, the meaning of the `shadow` property here is not entirely about `box-shadow`, it just indicates whether to enable shadow for the card, `box-shadow` is just one of the implementation details inside the component.